cypress/support/commands.js

```diff
@@ -26,6 +26,10 @@
 
 const imgUrl = 'https://static.productionready.io/images/smiley-cyrus.jpg';
 
+Cypress.Commands.add('findByPlaceholder', (placeholder) => {
+  cy.get(`[placeholder = "${placeholder}"]`);
+});
+
 Cypress.Commands.add('login', (email, username, password) => {
```

Comment:
It would be beneficial to add comments to your custom commands to describe what they do. This will make it easier for others (or future you) to understand the purpose of these commands.